Source/mainmenu.cpp

```diff
@@ -24,6 +25,10 @@ void mainmenu_refresh_music()
 
 void __stdcall mainmenu_change_name(int arg1, int arg2, int arg3, int arg4, char *name_1, char *name_2)
 {
+#ifdef PREFILL_PLAYER_NAME

```

Comment:
I left this part out of the rebase as it is never called (renaming only happens when your bnet login name isn't the same as the herro name, to prevent collisions)